sqlx-core/src/migrate/source.rs

```diff
@@ -54,8 +54,8 @@ pub struct ResolveError {
 
 // FIXME: paths should just be part of `Migration` but we can't add a field backwards compatibly
 // since it's `#[non_exhaustive]`.
-pub fn resolve_blocking(path: PathBuf) -> Result<Vec<(Migration, PathBuf)>, ResolveError> {
-    let mut s = fs::read_dir(&path).map_err(|e| ResolveError {
+pub fn resolve_blocking(path: &PathBuf) -> Result<Vec<(Migration, PathBuf)>, ResolveError> {
```

Comment:
`&PathBuf` is very uncommon because it coerces to `&Path`, which is generally a more useful type (because it can be directly created from `&str` and avoids an extra indirection). This should compile as well:

```suggestion
pub fn resolve_blocking(path: &Path) -> Result<Vec<(Migration, PathBuf)>, ResolveError> {
```

Reply:
Thanks for the review! Indeed, changed it now to `&Path`.